### PR TITLE
Icon, OG:Image and Readme

### PR DESCRIPTION
**Pull request by Gosia-Ras**
Hi guys, I updated the readme with a screenshot, description of what to do in the app and what technologies we use - let me know if you would add anything to it 🚀 

Also I added an icon to the page, it is the same as the icon next to main title on the page. 

The OG:Image is done in Canva, we can always change it later when we have the time ⏲️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+### API
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+This project uses API provided by [**The Movie Database**](https://www.themoviedb.org/).
 
-### `npm run build`
+### ➡️ Technologies
```
Comment (olachrzan) [inline README.md]: To me, the emoji arrows make the Technologies, Methodologies and How to use... look like they are part of the API.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
-In the project directory, you can run:
+## Description
 
-### `npm start`
+This **Movies Browser** app was created as a final group project for the ***Frontend Developer*** course by **YouCode**.
+You can use the app to browse and search for movies and people, as well as display the more detailed descriptions of them.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+### API
```
Comment (olachrzan) [inline README.md]: Maybe we don't do h3? I think if we leave h2 everywhere except title it will be fine 😃

File: public/favicon.svg [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="transparent" xmlns="http://www.w3.org/2000/svg">
+<rect width="40" height="40" x="0" y="0" fill="transparent" />
+<path d="M26.8774 24.9939L32.9858 27.4906C34.1458 27.9639 35.4158 27.1122 35.4158 25.8589V14.7939C35.4158 13.5406 34.1458 12.6872 32.9858 13.1606L26.8774 15.6572" stroke="silver" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M10.5068 31.9014H20.9552C24.5952 31.9014 26.8785 29.5297 26.8785 26.1714V14.4797C26.8785 11.1214 24.6052 8.74969 20.9552 8.74969H10.5068C6.85516 8.74969 4.5835 11.1214 4.5835 14.4797V26.1714C4.5835 29.5297 6.85516 31.9014 10.5068 31.9014Z" stroke="silver" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (olachrzan) [inline public/favicon.svg:4]: I'm thinking of changing the color to black to match the website title. What do you think? 🤔

File: public/index.html [hunk 4]
```diff
@@ -1,41 +1,42 @@
...
+    <link rel="icon" href="%PUBLIC_URL%/favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Movies Browser created using create-react-app"
+    />
+    <meta
+      property="og:image"
+      content="https://photos.app.goo.gl/iuSSmpZxN6PVwQGw9"
```
Comment (olachrzan) [inline public/index.html]: We have a png file, so we can use it 😉

File: README.md [hunk 5]
```diff
@@ -1,38 +1,46 @@
-# Movies Browser
+# **Movies Browser** 🎥
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<p align="center"><img src="https://raw.githubusercontent.com/olachrzan/movies-browser/Icon%26OG-Image/public/app-screenshot.jpg" alt="App layout screenshot"></p>
```
Comment (olachrzan) [inline README.md:4]: We can think about adding table of contents so that everyone can quickly find interesting content  😃

**Review by olachrzan — COMMENTED** [review]
Nice work, it's great to have the readme ready 😄

File: README.md (hunk 5 above)
Comment (Gosia-Ras) [reply README.md:4]: Good idea! Will do that

File: README.md (hunk 2 above)
Comment (Gosia-Ras) [reply README.md]: Yeah, looks better with h2 :rocket

File: README.md (hunk 1 above)
Comment (Gosia-Ras) [reply README.md]: I converted them to h2 :)

**Pull request merged**